### ISSUE TITLE
Latest Version with Tool library failed

### ISSUE THREAD

**Issue by Ruderg** [MscrmTools/XrmToolBox#1297]
Dear all,
when installing the latest XRMToolbox Version v1.2023.9.67 I get the message "Tool library initialization failed". There is no possibility to load the Tool Library.
I installed the previous version and everything works. [Write your comment/feedback/issue here]

---
- Connection Controls Version: 1.2023.6.56
- XrmToolBox Version: 1.2023.9.67

**Comment by Ruderg** [comment]
> Are you using a proxy on your network?

Yes, and I just changed the proxy settings to "Use default browser settings" and it worked.
Thanks!

**Comment by dornell2020** [comment]
Hi,
when opening the latest XRMToolbox Version v1.2023.12.68 I get the message "Tool library initialization failed". There is no possibility to load the Tool Library. 

I just changed the proxy settings to "Use default browser settings" and it does not work. Any ideas?

**Comment by MscrmTools** [comment]
When setting proxy, there should a test running that tells you if the connection to XrmToolBox api works or not.

Did you see it?

**Comment by dubvelis** [comment]
I am currently experiencing the same error "Tool library initialization failed", in my case I am not behind any proxy. I also went directly on the browser and tried the ulr https://www.xrmtoolbox.com/_odata/plugins which is the one used by the tool and it is not working. So how to resolve this issue now? v1.2025.10.74

**Comment by MscrmTools** [comment]
Sorry for the inconvenience, odata feed has been removed by Microsoft and I did not anticipated the deprecation (which was some years ago).
A new unsigned version is available that fixes the issue. Signed version will be available only after August 16th
